Documentation/General/finalReport/appendix-neuralnetwork.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Neural Networks}
+
+One of the bigger issues we had to deal with in MoodCat was to classify songs
+to moods. The initial plan was to create a neural network that could do this.
+
+Some of the MoodCat team members have followed a course about computational intelligence,
+in which different ways to use learning algorithms to solve different kinds of problems.
+One widely used technique to classify objects by using the objects' features as inputs
+is the use of neural networks.
+
+Neural networks consist of an input layer, an output layer and a number of hidden layers, 
+and each layer consists of a number of nodes.
+
+The nodes in the input layer are fed with 'features', if we take classifying songs as example
+this would be all kinds of information of the songs that can be represented as some form
+of numeric metrics. The amount of BPM of a song could be a metric, or the overall loudness
```

Comment:
'Different metrics could be the amount of BPM, the overall loundess or the tonality of songs.'